docs/guides/qiskit-code-assistant-jupyterlab.mdx

```diff
@@ -110,14 +110,14 @@ The service can sometimes take a few seconds to return a suggestion, you can see
 
 Jupyterlab also includes a traditional suggestion context menu. Use the `Tab` key to run and display the context menu.
 
-The context menu includes suggestions from JupyterLab in addition to the Qiskit Code Assistant suggestions. The context menu also sanitizes and trims the suggestions, making it less useful for reviewing the code suggestion before inserting it.
+The context menu includes suggestions from JupyterLab in addition to Qiskit Code Assistant suggestions. The context menu also sanitizes and trims the suggestions, making it less useful for reviewing the code suggestion before inserting it.
```

Comment:
```suggestion
The context menu includes suggestions from JupyterLab in addition to the Qiskit Code Assistant suggestions. The context menu also sanitizes and trims the suggestions, making it less useful for reviewing the code suggestion before inserting it.
```

Reply:
For some reason I don't think this one makes as much sense to add back in "the". I'll just rearrange it to eliminate the issue :)